WordPress/Classes/ViewRelated/System/Floating Create Button/WPTabBarController+CreateButton.swift

```diff
@@ -0,0 +1,29 @@
+import Gridicons
+
+extension WPTabBarController {
+
+    private enum Constants {
+        static let padding: CGFloat = -16
+        static let heightWidth: CGFloat = 56
+    }
+
+    @objc func addFloatingButton() {
+        let button = FloatingActionButton(image: Gridicon.iconOfType(.create))
+        button.accessibilityLabel = NSLocalizedString("Create", comment: "Accessibility label for create floating action button")
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.addTarget(self, action: #selector(showPostViewController(_:)), for: .touchUpInside)
+
+        view.addSubview(button)
+
+        NSLayoutConstraint.activate([
+            button.bottomAnchor.constraint(equalTo: tabBar.topAnchor, constant: Constants.padding),
+            button.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: Constants.padding),
+            button.heightAnchor.constraint(equalToConstant: Constants.heightWidth),
+            button.widthAnchor.constraint(equalToConstant: Constants.heightWidth)
+        ])
+    }
+
+    @objc func showPostViewController(_ sender: UIView) {
+        showPostTab()
```

Comment:
minor thing: once this is all done, since it's not gonna be a 'tab' anymore, maybe the name 'showPostTab' can be changed to something else

Reply:
> maybe the name 'showPostTab' can be changed to something else

Absolutely! This method will show the bottom sheet in the next set of changes anyway.